### PR TITLE
ARUHA-499 Use max_wait for nakadi requests

### PR DESCRIPTION
**Pull request by antban**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 from setuptools import setup
 from setuptools.command.test import test
 
-if sys.version_info < (3, 4, 0):
+if sys.version_info < (3, 5, 0):
     sys.stderr.write('FATAL: end2end needs to be run with Python 3.4+\n')
```
Comment (lmontrieux) [inline setup.py]: I guess this should now read "[...] with Python 3.5+"?

Comment (antban) [reply setup.py]: @lmontrieux yep(

**Comment by v-stepanov** [comment]
for me it's fine 👍

**Comment by lmontrieux** [comment]
Good for me as well :shipit:

**Comment by v-stepanov** [comment]
BTW, my current problem with end2end is that it is needed to be configured after deploy by calling some endpoint and providing some json. Can it somehow have some default values that will allow not to reconfigure it each time after re-launch of instance?

**Comment by antban** [comment]
@v-stepanov Yes, it is possible, because the configuration file is created within our appliance. We should update our appliance and deploy it.

**Pull request merged**